buildSrc/src/main/groovy/io.micronaut.build.internal.nats-module.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'io.micronaut.build.internal.module'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compileOnly 'com.google.code.findbugs:jsr305' // for "warning: unknown enum constant When.MAYBE"
+
+    testRuntimeOnly 'ch.qos.logback:logback-classic:1.2.7'
+}
+
+test {
+    useJUnitPlatform()
+
+    jvmArgs '-Duser.country=US'
+    jvmArgs '-Duser.language=en'
+    failFast = true
```

Comment:
:x: this should probably not be pushed as is. On CI you want all tests to be executed.